### PR TITLE
Beaver Builder Cards Enhancement

### PR DESCRIPTION
**Pull request by bhushanb20**
* PHPCS errors and warnings fixed
---------------------------------------------------------------------------------
Version 1.0.1 -

- Change: Old: Beaver Builder Bootstrap Cards | New: Beaver Builder Cards
- Update: Added image alt tag.
- Card image source enhancement: Media / URL.
- Update: Textarea to Text Editor for more flexibility.
- Fixed: PHP Notice Errors.

### CODE REVIEW

File: bb-bootstrap-cards-module/bb-bootstrap-cards-module.php [hunk 1]
```diff
@@ -173,12 +352,37 @@ public function get_alt()
...
                     'title'         => __('Select Card Image', 'bb-bootstrap-cards'), // Section Title
                     'fields'        => array( // Section Fields
+                        
+                        'cards_photo_source'  => array(
+                            'type'          => 'select',
+                            'label'         => __('Photo Source', 'bb-bootstrap-cards'),
+                            'default'       => 'library',
+                            'options'       => array(
+                                'library'       => __('Media Library', 'bb-bootstrap-cards'),
+                                'url'           => __('URL', 'fl-builder')
```
Comment (Nikschavan) [inline bb-bootstrap-cards-module/bb-bootstrap-cards-module.php]: Check the correct text-domain

File: bb-bootstrap-cards-module/bb-bootstrap-cards-module.php [hunk 2]
```diff
@@ -173,12 +352,37 @@ public function get_alt()
...
                         'photo'         => array(
                             'type'          => 'photo',
                             'label'         => __('Photo', 'bb-bootstrap-cards'),
                             'show_remove'   => true
                         ),
 
+                        'cards_photo_url'     => array(
+                            'type'          => 'text',
+                            'label'         => __('Photo URL', 'bb-bootstrap-cards'),
+                            'placeholder'   => __( 'http://www.example.com/my-photo.jpg', 'fl-builder' )
```
Comment (Nikschavan) [inline bb-bootstrap-cards-module/bb-bootstrap-cards-module.php]: Check the text-domain

**Review by Nikschavan — CHANGES_REQUESTED** [review]

**Comment by Nikschavan** [comment]
@bhushanb20 The PHPCS requirements are also required to be fixed before we merge this.

File: bb-bootstrap-cards-module/bb-bootstrap-cards-module.php (hunk 1 above)
Comment (bhushanb20) [reply bb-bootstrap-cards-module/bb-bootstrap-cards-module.php]: I have updated text domain to "bb-bootstrap-cards"

File: bb-bootstrap-cards-module/bb-bootstrap-cards-module.php (hunk 2 above)
Comment (bhushanb20) [reply bb-bootstrap-cards-module/bb-bootstrap-cards-module.php]: I have updated text domain to "bb-bootstrap-cards"

**Review by Nikschavan — APPROVED** [review]

**Pull request merged**